apollo-core/src/main/java/com/ctrip/framework/foundation/internals/provider/DefaultApplicationProvider.java

```diff
@@ -86,7 +86,15 @@ private void initAppId() {
       return;
     }
 
-    // 2. Try to get app id from app.properties.
+    //2. Try to get app id from OS environment variable
+    m_appId = System.getenv("APOLLO_APP_ID");
```

Comment:
why not just `APP_ID`?

Reply:
I just think `APOLLO_APP_ID ` won't conflict with user's own env variables. `APP_ID` still keep with `app.id`, this would be more friendly. OK, I will change to `APP_ID`.